### DISCUSSION TITLE
Installing ETNA in Anaconda

### DISCUSSION THREAD

**Discussion by DimartGit** [etna-team/etna#502] [Q&A] [answered]
Hi everybody!  
I'm trying to install **ETNA** in **Anaconda**. Having pure just installed Anaconda, I've greated new virt environment. The first command in that env is _pip install "etna[all]"_. The result is:
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
tinkoff-investments 0.2.0b58 requires protobuf<5.0.0,>=4.21.6, but you have protobuf 3.20.3 which is incompatible.

Obviously, the **tinkoff-investments** is a part of **ETNA**. So, how it could happened that the ERROR have been raised and how to solve the issue?

Thanks in advance for your advise.

**Comment by d-a-bunin** [comment]
Hi! Dependency`tinkoff-investments` isn't a part of `etna` package. It seems like you have a conflict between `tinkoff-investments` package and `etna` package on version of protobuf.

I suspect you don't really need `tinkoff-investments` package, in that case you could try to remove it to solve the problem.

**Comment by DimartGit** [comment]
Thanks a lot for your answer. 
I guess that **tinkoff-investments** is a part of **etna** because of result of _pip show tinkoff-investments_ cleary shows that **tinkoff-investments** is installed in the virt environment after the only command _pip install "etna[all]"_ on just created virt environment.
So, I cann't make an assumption why the conflict of **protobuf** version is up inside the whole etna package.

**Comment by egoriyaa** [comment]
Try to install `etna` in new empty env.

**Comment by DimartGit** [comment]
It was my first attempt to solve the issuue. I've got the same result. 
Futhemore, I can see the same behavior on stand alone python installation without Anaconda

**Comment by egoriyaa** [comment]
Looks strange.
1. Try to run `pip list` before and after `etna` installation
2. Also do you see `tinkoff-investments` in logs of `etna` installation?

**Comment by DimartGit** [comment]
1. There is **tinkoff-investments 0.2.0b58** in the results of _pip list_ 
2. The first inclusion in the logs is here: "The following dependency conflicts.
tinkoff-investments 0.2.0b58 requires protobuf<5.0.0,>=4.21.6, but you have protobuf 3.20.3 which is incompatible."

**Comment by egoriyaa** [comment, marked as answer]
As I understand, `tinkoff-investments` is already in your env before `etna` installation. So, that's an answer.

**Reply by DimartGit** [reply]
Thanks a lot for your support. I've just found were it was hided.
